docs/content/integrations/dbt/reference.mdx

```diff
@@ -537,20 +537,37 @@ Note that Dagster allows the optional specification of a [`code_version`](/conce
 
 ## Loading dbt tests as asset checks
 
-<Note>
-  <strong>Dagster asset checks are currently an experimental feature. </strong>{" "}
-  To provide feedback, join our{" "}
-  <a href="https://github.com/dagster-io/dagster/discussions/16527">
-    GitHub discussion
-  </a>{" "}
-  to share your use case with Dagster asset checks and dbt.
-</Note>
+Dagster loads your dbt tests as [asset checks](/concepts/assets/asset-checks).
+
+### Indirect selection
+
+Dagster uses [dbt indirect selection](https://docs.getdbt.com/reference/global-configs/indirect-selection) to select dbt tests. By default, Dagster won't set `DBT_INDIRECT_SELECTION` so that the set of tests selected by Dagster is the same as the selected by dbt. When required, Dagster will override `DBT_INDIRECT_SELECTION` to `empty` in order to explicitly select dbt tests. Examples:
+
+- materializing dbt assets and excluding their asset checks
+- executing dbt asset checks without materializing their assets
+
+### Singular tests
+
+Dagster will load both generic and singular tests as asset checks. In the event that your singular test depends on multiple dbt models, you can use dbt metadata to specify which Dagster asset it should target. These fields can be filled in as they would be for the dbt [ref function](https://docs.getdbt.com/reference/dbt-jinja-functions/ref).
+
+```yaml
+meta:
+  dagster:
+    ref:
+      name: ...
+      version: ...
+      package_name: ...
+```
+
+This requires dbt version 1.6 or later.
```

Comment:
Do we need to specify this is dbt Core? Also, what exactly requires dbt 1.6 - the ability to specify this metadata? Something else?

Reply:
cc @rexledesma - is "Manifest metadata includes project_name" the relevant bit from https://docs.getdbt.com/docs/dbt-versions/core-upgrade/upgrading-to-v1.6?

Reply:
Yeah, that's the relevant bit.